examples/google-gangway-example.yaml

```diff
@@ -0,0 +1,82 @@
+
```

Comment:
Is there a reason why this should be unique to Google?

Reply:
No, it isn't. I must've added it by accident. Will remove.